model/train.py

```diff
@@ -161,22 +161,48 @@ def compute_weighted_accuracy(logits, targets, weights=None):
     normalizing_factor = weights.sum()
 
   return loss.sum(), normalizing_factor
+  
+  
+def compute_sentence_accuracy(logits, targets, weights=None):
+  """Compute sentence accuracy for log probs and targets.
```

Comment:
For the style guide, docstrings should be descriptive-style rather than imperative-style.  (https://google.github.io/styleguide/pyguide.html#383-functions-and-methods)
```suggestion
  """Computes sentence accuracy for log probs and targets.
```